### PR TITLE
Add OpenDyslexic font as alternative user setting

### LINKED ISSUE #1606: Add OpenDyslexic font as alternative user setting
[From the forum](https://groups.google.com/d/msg/mitappinventortest/KFkF0ZfUI6g/rhRAIFUJBQAJ): Adding the ability to use a font such as OpenDyslexic to improve the readability of the web interface would benefit app inventors. I don't think we want to make it the default, but having an option to switch it on (and store the state in UserSettings) would make the platform more accessible.

### PR DESCRIPTION
**Pull request by bartmathijssen**
Resolves #1606

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

File: appinventor/appengine/war/Ya.css [hunk 1]
```diff
@@ -29,15 +40,15 @@ body, html {
...
   /* TODO(opensource): Change to commented version below once we're ready */
   background: url(images/squairy_light.png);
-  font-family: Roboto, Helvetica, Arial, sans-serif;
 }
 
-table td,
-select {
-  font-family: Roboto, Helvetica, Arial, sans-serif;
+.dyslexic * {
+  font-family: OpenDyslexic, Helvetica, Arial, sans-serif;
```
Comment (ewpatton) [inline appinventor/appengine/war/Ya.css]: Include Roboto as a backup font here so if for some reason OpenDyslexic doesn't load we will fall back to the existing font.

File: appinventor/blocklyeditor/media/ai2blockly.css [hunk 2]
```diff
@@ -36,3 +36,8 @@
 .blocklyZoomButton:hover {
   cursor: pointer;
 }
+
+.dyslexic .blocklyText,
+.dyslexic .blocklyFieldParameter {
+  font-family: OpenDyslexic, Helvetica, Arial, sans-serif;
+}
```
Comment (ewpatton) [inline appinventor/blocklyeditor/media/ai2blockly.css]: Missing newline at end of file.

**Comment by bartmathijssen** [comment]
@ewpatton I removed the unused OpenDyslexic files, they must have been there from when I started working from #1639.  I also resolved the other issues that were present.

**Pull request merged**